### PR TITLE
Update appveyor powershell names & scoop git url

### LINKED ISSUE #2824: [CI] Update to PowerShell 7
I've noticed that the CI is currently using PowerShell 5 and PowerShell Core 6 while PowerShell 7 is the current recommended version. I think we need to add PowerShell 7 into the CI process.

I'm also pretty sure PowerShell Core 6 is EOL, so once we have PowerShell 7 working we would probably want to remove it.

### PR DESCRIPTION
**Pull request by tech189**
Closes #2824

### CODE REVIEW

**Review by stevehipwell — COMMENTED** [review]
/lgtm

**Review by stevehipwell — APPROVED** [review]
/lgtm

**Comment by rashil2000** [comment]
@stevehipwell is there a way to test this before merging into master (just to be sure it doesn't break anything)?

**Comment by stevehipwell** [comment]
I don't think there is an easy way.

File: appveyor.yml [hunk 1]
```diff
@@ -14,14 +14,14 @@ environment:
   innounp: '%scoop_helpers%\innounp\innounp.exe'
   matrix:
     - PowerShell: 5
-    - PowerShell: 6
+    - PowerShell: Current
```
Comment (pratikpc) [inline appveyor.yml:17]: Can you share an official link which mentions this is the correct behaviour to get latest Powershell?

Comment (rashil2000) [reply appveyor.yml:17]: The matrix just mentions the name, this part is not responsible for pulling in PowerShell

Comment (pratikpc) [reply appveyor.yml:17]: So would using PowerShell 5 for example lead to it also using PowerShell's newest version? That does sound a bit weird to me.

Not very well versed with AppVeyor.

Comment (stevehipwell) [reply appveyor.yml:17]: @pratikpc the names here are just matrix keys so the job gets run once for `powershell` and once for `pwsh`, changing the name stops the misconception that we're using PS 6 as AppVeyor keeps the version of `pwsh` current (so v7 ATM).

Comment (pratikpc) [reply appveyor.yml:17]: @stevehipwell ps indicates PowerShell 5 while pwsh indicates PowerShell's newest version right?

**Review by pratikpc — APPROVED** [review]

**Comment by rashil2000** [comment]
Merging this now, fingers crossed 🤞


Thank you for the contribution @tech189 :)

**Pull request merged**